sjb/config/test_cases/test_branch_origin_web_console.yml

```diff
@@ -2,14 +2,44 @@
 parent: 'common/test_cases/origin_built_release.yml'
 extensions:
   actions:
+    - type: "script"
+      title: "determine binary and image version"
+      repository: "origin"
+      script: |-
+        case $PULL_BASE_REF in
+        enterprise-3.[8-9])
+          echo "IMAGE_VERSION=v3.9" > IMAGE_VERSION_VAR
+          ;;
+        enterprise-3.7)
+          echo "IMAGE_VERSION=v3.7" > IMAGE_VERSION_VAR
+          ;;
+        enterprise-3.6)
+          echo "IMAGE_VERSION=v3.6.1" > IMAGE_VERSION_VAR
+          ;;
+        enterprise-3.[2-5])
+          echo "IMAGE_VERSION=${PULL_BASE_REF/enterprise-3/v1}.1" > IMAGE_VERSION_VAR
+          ;;
+        enterprise-3.1)
+          echo "IMAGE_VERSION=v1.1.6" > IMAGE_VERSION_VAR
+          ;;
+        *)
+          echo "IMAGE_VERSION=latest" > IMAGE_VERSION_VAR
```

Comment:
This looks like any newer branches like enterprise-3.10, enterprise-3.11, etc. will use latest. I'd rather write it in a way that will work for future versions without needing to update the job.